specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json

```diff
@@ -0,0 +1,1912 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-01",
+    "title": "Billing benefits RP",
+    "description": "Azure Benefits RP let users create and manage benefits like savings plan."
+  },
```

Comment:
Please respond to the email thread providing evidence of a CVP approval to not be on RPaaS

Reply:
Got an approval offline